tests/testthat/_snaps/update_ard.md

```diff
@@ -1,14 +1,3 @@
-# update_ard_fmt_fn()
-
-    Code
-      update_ard_fmt_fn(ard_continuous(ADSL, variables = AGE), stat_names = c("mean",
-        "sd"), fmt_fn = -8L)
-    Condition
-      Error in `update_ard_fmt_fn()`:
-      ! The value in `fmt_fn` cannot be converted into a function.
-      i Value must be a function, a non-negative integer, or a formatting string, e.g. "xx.x".
-      * See `?cards::alias_as_fmt_fn()` for details.
```

Comment:
this was informative imo

Reply:
it is back now! (related to your other comment)